grid-cli/cmd/deploy_vm.go

```diff
@@ -36,6 +36,10 @@ var deployVMCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		env, err := cmd.Flags().GetStringToString("env")
```

Comment:
Where do we validate against the value of env?

Reply:
We don't, what validation should be set on a map of arbitrary keys and values?

Reply:
Ah! I thought it was threefold grid env not env vars